### PR TITLE
updated specs

### PR DESCRIPTION
**Pull request by vishwanathjadhav**
On RHEL6 grubby cmd path is different.
https://github.com/RedHatInsights/insights-core/issues/2578

Signed-off-by: vishawanathjadhav <vjadhav@redhat.com>

### CODE REVIEW

**Comment by vishwanathjadhav** [comment]
@xiangce - Please review the change.

File: insights/specs/default.py [hunk 1]
```diff
@@ -350,7 +350,7 @@ def dumpdev(broker):
     grub2_cfg = simple_file("/boot/grub2/grub.cfg")
     grub2_efi_cfg = simple_file("boot/efi/EFI/redhat/grub.cfg")
     grubby_default_index = simple_command("/usr/sbin/grubby --default-index")  # only RHEL7 and updwards
-    grubby_default_kernel = simple_command("/usr/sbin/grubby --default-kernel")  # RHEL6 and updwards
+    grubby_default_kernel = simple_command("/sbin/grubby --default-kernel")  # RHEL6 and updwards
```
Comment (xiangce) [inline insights/specs/default.py:353]: As talked in IRC, you'd open an INSPEC card for this Spec Change at first.

And FYI, please use `first_of` to support all the RHEL 6/7/8.

```
[root@liuxc-rhel6-sat6 ~]# uname -r
2.6.32-754.9.1.el6.x86_64
[root@liuxc-rhel6-sat6 ~]# which grubby
/sbin/grubby

[root@liuxc-rhel7-sat6 ~]# uname -r
3.10.0-957.12.1.el7.x86_64
[root@liuxc-rhel7-sat6 ~]# which grubby
/usr/sbin/grubby

[root@team-jupyter-server ~]# uname -r
4.18.0-147.5.1.el8_1.x86_64
[root@team-jupyter-server ~]# which grubby
/usr/sbin/grubby
```

Comment (vishwanathjadhav) [reply insights/specs/default.py]: @xiangce - Done changes prelease review

File: insights/specs/default.py [hunk 2]
```diff
@@ -350,7 +350,10 @@ def dumpdev(broker):
     grub2_cfg = simple_file("/boot/grub2/grub.cfg")
     grub2_efi_cfg = simple_file("boot/efi/EFI/redhat/grub.cfg")
     grubby_default_index = simple_command("/usr/sbin/grubby --default-index")  # only RHEL7 and updwards
-    grubby_default_kernel = simple_command("/usr/sbin/grubby --default-kernel")  # RHEL6 and updwards
+    grubby_default_kernel = first_of([
+                                        simple_command("/sbin/grubby --default-kernel"),
+                                        simple_command("/usr/sbin/grubby --default-kernel")
+                                    ])
```
Comment (bfahr) [inline insights/specs/default.py:356]: @vishwanathjadhav you also need to modify the archive path in `insights.specs.insights_archive` to match this change.

Comment (vishwanathjadhav) [reply insights/specs/default.py:356]: There wont be any change in the insights_archive.py
~~~
grubby_default_kernel = simple_file("insights_commands/grubby_--default-kernel")
~~

Comment (xiangce) [reply insights/specs/default.py:356]: Yes, the mangle results are `insights_commands/grubby_--default-kernel`  for both items.

File: insights/specs/default.py (hunk 1 above)
Comment (xiangce) [reply insights/specs/default.py]: @vishwanathjadhav - I made a mistake here, in RHEL 7/8, the `/sbin/` is also a symbol link to the `/usr/sbin/` by default. So, your first version is OK. I'm sorry about that. Could you please revert it back?

**Pull request merged**